integration/forks/test_helpers

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+build_images() {
+  docker build -t geth ./geth
+	docker build -t smartcontract/chainlink ../../
+}
+
+start_network() {
+	echo "Starting Network. See logs/{chain_[12],geth}.log for logs"
+	( docker-compose up > /dev/null ) &
+}
+
+tear_down() {
+	docker-compose down
+	# Remove CL database
+	sudo rm -rf tmp
+}
+
+login() {
+  until docker exec -it forks_chainlink chainlink admin login -f /run/secrets/apicredentials
+  do
+      sleep 0.1
+  done
+}
+
+create_job() {
+  ETH_LOG_JOB=`cat fixtures/eth_log_job.json`
+	# wait for CL to start
+	search_chainlink_logs 'Listening and serving HTTP on port 6688'
```

Comment:
Again, could use `waitFor` here also. It's also a bit strange that we essentially have two checks waiting for the node to start, run back to back. One in `create_job` and one in `login`.